### ISSUE TITLE
scoring-AQ10

### ISSUE THREAD

**Issue by SDOsmany** [NDCLab/instruments#8]
**Scoring Instructions:**

>Only 1 point can be scored for each question. Score 1 point for Definitely or
Slightly Agree on each of items 1, 7, 8, and 10. Score 1 point for Definitely or Slightly
Disagree on each of items 2, 3, 4, 5, 6, and 9.

Link to instrument [here](https://drive.google.com/drive/u/5/folders/16T-3tH3xkdz1x7rvoGgh_hX9ZdYldKGt).

**Questions**:

1.Are there any subscales that the lab wants to use?

**Done Defined:**

-AQ10 added to the scoring script on dev-script (OP/FS)
-AQ10 added to dev-uploads (PDF, REDCap zip, readme) (JA)

**Comment by jessb0t** [comment]
@georgebuzzell  Please provide guidance on questions above.  Thanks!

**Comment by georgebuzzell** [comment]
1. for any survey, please always score all scales/subscales.
2. is AQ10 a sum score or average score? if sum score, our current plan is to not score if even one value is missing. in the future, we may consider adding alternative methods for computing scores on items requiring sums if an item is missing, but for now, we are not scoring. Note that this should be pretty rare. 

but again, my answer to number 2 depends on if the aq10 is a sum or average score

**Comment by jessb0t** [comment]
@georgebuzzell: It seems that the AQ-10 is a sum score, but it's a bit special where only certain answers get one point and then those are added together.  The instrument says:
>  If the individual scores 6 or above, consider referring them for a specialist diagnostic assessment.

**Comment by jessb0t** [comment]
@SDOsmany 
Based on our convo today, let's proceed with the following assumptions:
1. Simple sum score (no subscales).
2. If any data is missing, we don't calculate a score.

We will await confirmation from @georgebuzzell of these assumptions before we close this issue, but will code this way in the script for now. :)

**Comment by jessb0t** [comment]
@georgebuzzell Please note that we are awaiting your final confirmation here.  Thanks!

**Comment by georgebuzzell** [comment]
@jessb0t I think there is maybe a slight misunderstanding somewhere here?

My understanding is that the instrument should be scored as one sum score, across all 10 items, but just that some of the items get a value of 1 for Definitely or Slightly Agree, whereas some get a value of 1 for Definitely or Slightly
Disagree.

The last comment about a score above 6 is for referring for clinical services and not needed for us here.

SCORING: Only 1 point can be scored for each question. Score 1 point for Definitely or
Slightly Agree on each of items 1, 7, 8, and 10. Score 1 point for Definitely or Slightly
Disagree on each of items 2, 3, 4, 5, 6, and 9. If the individual scores 6 or above, consider
referring them for a specialist diagnostic assessment.

**Comment by jessb0t** [comment]
@georgebuzzell Yes, I believe we are on the same page about how the maximum number of 'points' is 10.  Could you please confirm that we should not calculate a score if any datapoint is missing?

**Comment by georgebuzzell** [comment]
@jessb0t ah, ok; yes! confirming not to compute if any value is missing, given that this refelcts a sum score.

in the future, we want to add in a modification to compute sum scores when values are missing, but not for now.

**Comment by jessb0t** [comment]
Closing with #13.
